cmd/start.go

```diff
@@ -32,12 +32,18 @@ func init() {
 		},
 	}
 
-	cmd.PersistentFlags().StringVar(&opts.ListenAddr, "listen", opts.ListenAddr, "Address to listen on for metrics")
-	cmd.PersistentFlags().StringVar(&opts.Kubeconfig, "kubeconfig", opts.Kubeconfig, "Kubeconfig file to access a remote cluster (testing only)")
+	cmd.PersistentFlags().StringVar(&opts.ListenAddr, "listen", opts.ListenAddr, "Address to listen on for metrics.")
+	cmd.PersistentFlags().StringVar(&opts.Kubeconfig, "kubeconfig", opts.Kubeconfig, "Kubeconfig file to access a cluster where the CVO is reconciling manifests.")
 	cmd.PersistentFlags().StringVar(&opts.NodeName, "node-name", opts.NodeName, "kubernetes node name CVO is scheduled on.")
 	cmd.PersistentFlags().BoolVar(&opts.EnableAutoUpdate, "enable-auto-update", opts.EnableAutoUpdate, "Enables the autoupdate controller.")
 	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The Openshift release image url.")
 	cmd.PersistentFlags().StringVar(&opts.ServingCertFile, "serving-cert-file", opts.ServingCertFile, "The X.509 certificate file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file unless you set --listen empty.")
 	cmd.PersistentFlags().StringVar(&opts.ServingKeyFile, "serving-key-file", opts.ServingKeyFile, "The X.509 key file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file unless you set --listen empty.")
+	cmd.PersistentFlags().StringVar(&opts.PromQLTarget.QueryCaBundleFile, "prometheus-query-ca-bundle-file", opts.PromQLTarget.QueryCaBundleFile, "The service CA bundle file containing one or more X.509 certificate files for validating certificates generated from the service CA for the respective PromQL target.")
+	cmd.PersistentFlags().StringVar(&opts.KubeconfigPromQLTargetCluster, "prometheus-query-kubeconfig", opts.KubeconfigPromQLTargetCluster, "The kubeconfig file to access a cluster where the PromQL query service resides.")
```

Comment:
We have used the `prometheus` string for the flags. But IMO we should use `metrics` instead of `prometheus`. It is not blocker for this PR to merge but a suggestion.

Reply:
Yes, I like the idea of it :+1: `prometheus` is more explicit of the intended use of the flag but `metrics` is more user friendly and conveys the intent better IMO. I have myself named some variables (now non-existing) in HyperShift `metricsServer` to explain the intent.

Current naming of the added flags:
```go
"metrics-ca-bundle-file"
"metrics-token-file",
"metrics-namespace"
"metrics-service"
"use-dns-for-services"
"metrics-url"
"hypershift"
```